bio/gatk/applybqsrspark/meta.yaml

```
@@ -0,0 +1,18 @@
+name: gatk ApplyBQSRSpark
+description: |
+  Run gatk ApplyBQSRSpark.
```

Maybe a bit more descriptive?
```suggestion
  ApplyBQSRSpark: Apply base quality score recalibration on Spark; uses output of the BaseRecalibrator tool.
```